languagetool-core/src/main/java/org/languagetool/Language.java

```diff
@@ -536,6 +536,9 @@ public void setPostDisambiguationChunker(Chunker chunker) {
   /**
    * Create a shared instance of JLanguageTool to use in rules for further processing
    * Instances are shared by Language
+   *
+   * Do not call this in a static block or to initialize a static field in rules or filters.
```

Comment:
Maybe mention the consequences, and also that configuring this with e.g. disableRules is also bad.
Can we recommend alternatives?